core/arch/interp.c

```diff
@@ -4196,6 +4223,12 @@ build_bb_ilist(dcontext_t *dcontext, build_bb_t *bb)
         }
 #endif
     }
+    else if (dcontext->forged_exception_addr == bb->start_pc) {
+        /* Might have been cleared by client_process_bb. */
```

Comment:
Qin also pointed out that drutil_expand_rep_string() is a legitimate thing that would cause this.  Hmm.